android/build.gradle

```diff
@@ -1,56 +1,57 @@
 buildscript {
-  repositories {
-    jcenter()
-    google()
-    maven {
-      url 'https://maven.google.com'
+    repositories {
+        jcenter()
+        maven {
+            url 'https://maven.google.com/'
+            name 'Google'
+        }
     }
-  }
 
-  dependencies {
-    classpath 'com.android.tools.build:gradle:3.0.0'
-  }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:+'
```

Comment:
we are experiencing some weird behaviours with this approach on our apps
can we keep `3.0.0`?

Reply:
we can revert it to 3.0.0, but can you check first if those weird behaviours still persist when we use like this